docs/messaging.rst

```diff
@@ -128,6 +128,10 @@ A message is defined by the following four-dictionary structure::
       # The actual content of the message must be a dict, whose structure
       # depends on the message type.
       'content' : dict,
+
+      # optional: buffers is a list of binary data buffers for implementations
+      # that support binary extensions to the protocol.
+      'buffers': list,
```

Comment:
`buffers` is raw data - on the websockets (and in node zmq), these end up being `Uint8Array`.